static/app/components/idBadge/teamBadge/index.tsx

```diff
@@ -8,6 +9,18 @@ import Badge, {BadgeProps} from './badge';
 function TeamBadge(props: BadgeProps) {
   const [team, setTeam] = React.useState<Team>(props.team);
 
+  React.useEffect(() => {
+    if (team === props.team) {
```

Comment:
i know this was copy pasted from the old class component implementation, but do we need both if statements here? would the second one be sufficient already?